pom.xml

```
@@ -69,10 +69,10 @@
       <artifactId>ionicons-api</artifactId>
     </dependency>
     <dependency>
-      <groupId>org.mockito</groupId>
-      <artifactId>mockito-core</artifactId>
-      <scope>test</scope>
-    </dependency>
+     <groupId>org.mockito</groupId>
+       <artifactId>mockito-core</artifactId>
+       <scope>test</scope>
+</dependency>
```

Don't need this change that only affects white space
```suggestion
      <groupId>org.mockito</groupId>
      <artifactId>mockito-core</artifactId>
      <scope>test</scope>
    </dependency>```